.gitignore

```diff
@@ -10,3 +10,4 @@ node_modules/
 *.ipynb*
 courses20.xml
 .coverage
+src/api/tests/Dockerfile
```

Comment:
This should not be getting ignored here. If you did this to stop from pushing some kind of change to the dockerfile, you should prevent that by only adding files you wish to change.

Reply:
Change has been discard

Reply:
This is still in the files changed